src/services/utilities.ts

```diff
@@ -436,6 +436,35 @@ namespace ts {
         return isInCommentHelper(sourceFile, position, /*predicate*/ undefined);
     }
 
+    export function isInJsxAttribute(sourceFile: SourceFile, position: number) {
+        let token = getTokenAtPosition(sourceFile, position);
+        if (token && (token.kind === SyntaxKind.JsxAttribute || token.kind === SyntaxKind.JsxSpreadAttribute)) {

```

Comment:
`JsxAttribute` and `JsxSpreadAttribute` are not a tokens, they are nodes, so do you have tests where you run into this?